encoder/vaapiencoder_base.cpp

```diff
@@ -341,6 +348,146 @@ static bool copyImage(uint8_t* destBase,
     return true;
 }
 
+static uint32_t get_bpp_in_1stplane_by_fourcc(uint32_t fourcc)
+{
+    uint32_t bpp;
+
+    switch (fourcc) {
+    case VA_FOURCC_NV12:
+    case VA_FOURCC_YV12:
+    case VA_FOURCC_I420:
+        bpp = 1;
+        break;
+
+    case VA_FOURCC_P010:
+    case VA_FOURCC_P016:
+        bpp = 2;
+        break;
+
+    default:
+        bpp = 1;
+        break;
+    }
+
+    return bpp;
+}
+
+static bool loadFrameToSurface(VAImage surface_image,uint8_t* dst,uint8_t* src,uint32_t fourcc)
```

Comment:
it convert I010->P010 and I420->NV12. copyImage doesnot.

the driver can convert I420->NV12 when input is I420.  and it does not support I010->I420.